docs/application/dotnet/guides/app-management/application-preference.md

```diff
@@ -0,0 +1,266 @@
+# Application Preferences
```

Comment:
Global comment: For new files we recommend to follow Sentence case for headings.
Please change to 

# Application preferences